.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: "\U0001F41E Bug Report"
+about: Did you find a bug?
+title: ''
+labels: 'Bug'
+assignees: ''
+
+---
+
+<!--
+  Hi there! Thank you for discovering and submitting an issue.
+
+  Before you submit this; let's make sure of a few things.
+  Please make sure the following boxes are ticked if they are correct.
+  If not, please try and fulfill these first.
+-->
+
+<!-- Checked checkbox should look like this: [x] -->
+- [ ] I am on the [latest](https://github.com/aiplanethub/genai-stack/releases/tag/0.2.0-alpha) GenAI version.
```

Comment:
Replace the url with `https://github.com/aiplanethub/genai-stack/releases/latest`, so we dont have to update it everytime a release is made.